api/pkg/filters/auth/keystone.go

```diff
@@ -22,10 +22,10 @@ import (
 	"time"
 
 	"github.com/emicklei/go-restful"
```

Comment:
can you please share the requirements and design document for this feature, thanks.